js/ui/tooltip/tooltip.js

```diff
@@ -24,6 +24,12 @@ const Tooltip = Popover.inherit({
             */
             showCloseButton: false,
 
+            /**
+            * @name dxTooltipOptions.enableBodyScroll
+            * @hidden
+            */
+            enableBodyScroll: true,
```

Comment:
shouldn't it be `false` by default to keep an old behavior in Sheduler appointment, slider tooltip and so on?

Reply:
In Sheduler appointment we set it as false, because it's justified there. But in tooltip slider and others there are a lot of use cases where it may be redundant.